<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JMD Video Compare UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelVideo1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video 1 (left side)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideo1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>380</width>
      <height>24</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVideo1Browse">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideo2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>380</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonVideo2Browse">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video 2 (right side)</string>
    </property>
   </widget>
   <widget class="Line" name="vline">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>3</width>
      <height>210</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>JMD Video Compare UI</string>
    </property>
   </widget>
   <widget class="Line" name="hlineHeader">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="hlineContent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>1000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelStartTimeVideo1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>215</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Time (HH:MM:SS)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStartTimeVideo1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>215</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDuration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>00:01:30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duration (HH:MM:SS)</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBoxVideo1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo1AddText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Overlay Text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo1Font">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideo1Text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>471</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo1AddTextBottom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo1AddTextTop">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>50</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo1AddTextMiddle">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Middle</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo1TextPosition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Text position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo2Font">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Font:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo2AddText">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Overlay Text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo2AddTextTop">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBoxVideo2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>120</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo2AddTextBottom">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bottom</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideo2TextPosition">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>90</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Text position:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVideo2Text">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>471</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxVideo2AddTextMiddle">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Middle</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>170</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output Video Options</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStartTimeVideo2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>215</y>
      <width>301</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelStartTimeVideo2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>215</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Time (HH:MM:SS)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFPS">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Framerate (FPS)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFPS">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelVideoCodec">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video Codec</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVideoCodec">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAudioCodec">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>280</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Audio Codec</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAudioCodec">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelBitrate">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>280</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bitrate (kbps)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBirate">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>310</y>
      <width>180</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>4000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOutputVideoDivider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>345</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Vertical Divider?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputVideoDividerWidth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>395</y>
      <width>210</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputVideoDivider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Width (in px - must be even number)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputVideoDividerColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>370</y>
      <width>210</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVideoDividerColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>395</y>
      <width>210</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentText">
     <string>white</string>
    </property>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>grey</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>980</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVideo1AddTextColor">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>180</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentText">
     <string>white</string>
    </property>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>grey</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelVideo1AddTextColor">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>180</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxVideo2AddTextColor">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentText">
     <string>white</string>
    </property>
    <item>
     <property name="text">
      <string>white</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>grey</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelVideo2AddTextColor">
    <property name="geometry">
     <rect>
      <x>855</x>
      <y>180</y>
      <width>40</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOutputAudioVideo1">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>360</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use audio from Video 1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxOutputAudioVideo2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>390</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Use audio from Video 2</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="hlineContent_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>1000</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputVideoFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output Video File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditOutputVideoFile">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>501</width>
      <height>24</height>
     </rect>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutputVideoBrowse">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputVideoType">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>440</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Video Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxOutputVideoType">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>440</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mkv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>flv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wmv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>webm</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>980</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="logoButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>168</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/JMDigital.png</normaloff>images/JMDigital.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>160</width>
      <height>35</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
